JSDemos/Demos/TabPanel/Overview/Angular/app/app.component.html

```diff
@@ -16,7 +16,7 @@
         <div class="tabpanel-item">
           <div
             *ngFor="let task of tabPanelItem.tasks"
-            [className]="getTaskClassName(task.priority)"
+            [class]="getTaskClassName(task.priority)"
```

Comment:
getTaskClassName() function returns several classes. The method name should have other name what displays the actual operation of the method. When we used [className] the name 'getTaskClassName' was correct.

Reply:
[d32bbc8576822962a199e7cde00e283894f357bb]